Dockerfile-dalton

```diff
@@ -23,10 +23,11 @@ COPY dalton.conf /opt/dalton/dalton.conf
 COPY rulesets /opt/dalton/rulesets
 COPY engine-configs /opt/dalton/engine-configs
 
-#install flowsynth
-ADD https://github.com/secureworks/flowsynth/raw/master/requirements.txt /opt/flowsynth/requirements.txt
-ADD https://github.com/secureworks/flowsynth/raw/master/src/flowsynth.py /opt/flowsynth/src/flowsynth.py
-RUN pip install -r /opt/flowsynth/requirements.txt
-RUN chmod +x /opt/flowsynth/src/flowsynth.py
+# install flowsynth
+RUN git clone https://github.com/secureworks/flowsynth.git /opt/flowsynth/
+RUN pip install /opt/flowsynth
+# When a packaged version of flowsynth is available on https://pypi.org, the
```

Comment:
couldn't you just add it to requirements.txt in that case?

Reply:
Yes, that would be even easier.